packages/ferium-bin/ferium-bin.pacscript

```diff
@@ -17,4 +18,10 @@ build() {
 
 install() {
   sudo install -Dm755 ferium "${STOWDIR}/${name}/usr/bin/${pkgname}"
+  # install completions
+  "${STOWDIR}/${name}/usr/bin/${pkgname}" complete bash | sudo install -Dm644 /dev/stdin "${STOWDIR}/${name}/usr/share/bash-completion/completions/ferium"
+  "${STOWDIR}/${name}/usr/bin/${pkgname}" complete fish | sudo install -Dm644 /dev/stdin "${STOWDIR}/${name}/usr/share/fish/vendor_completions.d/ferium.fish"
+  "${STOWDIR}/${name}/usr/bin/${pkgname}" complete zsh | sudo install -Dm644 /dev/stdin "${STOWDIR}/${name}/usr/share/zsh/site-functions/_ferium"
 }
+
+# vim:set ft=sh ts=2 sw=2 et:
```

Comment:
Should this be here?

Reply:
It's up to the maintainer. This line makes (neo)vim show the file with shell syntax highlighting, and a couple other things, which make sure that the file conforms to the styling that we use.